Gemfile

```diff
@@ -42,7 +46,8 @@ group(:omnibus_package) do
   gem "chef-provisioning", ">= 2.4.0"
   gem "chef-provisioning-aws", ">= 2.0"
   gem "chef-provisioning-azure", ">= 0.6.0"
-  gem "chef-provisioning-fog", ">= 0.20.0"
+  # As of October 2017, `chef-provisioning-fog` makes the Gemset unsolvable by bundler
+  # gem "chef-provisioning-fog", ">= 0.20.0"
```

Comment:
Just making sure everyone sees this.

Reply:
i agree, but #1413

Reply:
What's the solution then? `bundle install` runs for at least 20 minutes with this here, with no clear indication that it will ever actually finish.

Reply:
given this is a new thing, I don't think this is a major problem. We aren't removing something from an existing artefact that someone is reliant on.

Reply:
@thommay this is the main Gemfile so it would impact the omnibus releases, too :|

Reply:
urgh, right. This is why i should sleep more. We can't merge this, then.

Reply:
I'm cool to wait on merging and rebase later, but we're in a pretty rough situation with the bundle until this gets fixed--not sure we can even bring in a new chef-client at this point :(

Reply:
well, after letting run for an hour, I finally got:

Reply:
ah, half of the cheffish-13-ing of chef-provisioning-fog was done in https://github.com/chef/chef-provisioning-fog/pull/219/files

master is still busted by some hard-to-diagnose problems with a few of the plugins.  if those were fixed, then you could roll back to the old fog-google pin and then release a cheffish-13 compatible version.

the big problem is the cheffish 5.0 break turning the MergedConfig into merged Mashes instead of merged Hashes, which broke serialization to symbols since they get converted to strings.

Reply:
This should be solved with a rebase on to master…